TAF/utils/scripts/snap/deploy-services.sh

```diff
@@ -1 +1,3 @@
 #!/bin/sh
+
+logger "[snap] deploy-services: $@"
```

Comment:
Same comment as above, and really applies to all of these shell scripts which just call `logger`.

Reply:
This script is not used and I've deleted it.